packages/checkbox/src/index.mts

```diff
@@ -82,7 +82,7 @@ export default createPrompt(
 
     const bounds = useMemo(() => {
       const first = items.findIndex(isSelectable);
-      const last = items.findLastIndex(isSelectable);
+      const last = items.length - 1 - [...items].reverse().findIndex(isSelectable);
```

Comment:
Ugly... But works until Node 16 is EOL (well it is, but just since a few days ago) and TS catch up.